src/core/ext/filters/client_channel/lb_policy.h

```diff
@@ -202,8 +202,7 @@ class LoadBalancingPolicy : public InternallyRefCounted<LoadBalancingPolicy> {
     /// by the client channel.
     virtual void UpdateState(grpc_connectivity_state state,
                              grpc_error* state_error,
-                             UniquePtr<SubchannelPicker> picker) {
-      std::move(picker);  // Suppress clang-tidy complaint.
+                             UniquePtr<SubchannelPicker>) {
```

Comment:
We can replace the body of this method (including the braces) with just `GRPC_ABSTRACT`.

Reply:
OK. Done.